### PR TITLE
DOC: Added examples to maximum_position in ndimage

### PR DESCRIPTION
**Pull request by nwjenkins**
#### Reference issue
Example: addresses part of issue #7168

#### What does this implement/fix?
Added a few examples to `maximum_position` in `ndimage` where there were none before

#### Additional information
This is my first pull request, so I am just learning the process. Please let me know if there is anything I should be doing better in the future (or now!). Thanks!

### CODE REVIEW

File: scipy/ndimage/measurements.py [hunk 1]
```diff
@@ -1205,6 +1205,33 @@ def maximum_position(input, labels=None, index=None):
     --------
     label, minimum, median, maximum_position, extrema, sum, mean, variance,
     standard_deviation
+    
+    Examples
+    --------
+    >>> from scipy import ndimage
+    >>> import numpy as np
```
Comment (rgommers) [inline scipy/ndimage/measurements.py]: this import can be removed, it's implicit in all SciPy examples

File: scipy/ndimage/measurements.py [hunk 2]
```diff
@@ -1205,6 +1205,33 @@ def maximum_position(input, labels=None, index=None):
...
+    >>> a = np.array([[1, 2, 0, 0],
+    ...               [5, 3, 0, 4],
+    ...               [0, 0, 0, 7],
+    ...               [9, 3, 0, 0]])
+    >>> ndimage.maximum_position(a)
+    (3, 0)
+
+    Features to process can be specified using `labels` and `index`:
+
+    >>> lbl = np.matlib.repmat([0,1,2,3],4,1)
```
Comment (rgommers) [inline scipy/ndimage/measurements.py]: I'd prefer an explicit use of `np.array` here, like a few lines above. Clearer to the reader, and `np.matlib.repmat` is a function that's best avoided.

**Review by rgommers — CHANGES_REQUESTED** [review]
Thanks @nwjenkins, this looks good module two minor comments.

**Comment by rgommers** [comment]
One of the CI failures is actually real:
```
scipy.ndimage.maximum_position
------------------------------
File "build/testenv/lib/python3.6/site-packages/scipy/ndimage/measurements.py", line 1225, in maximum_position
Failed example:
    ndimage.maximum_position(a, lbl, 1)
Expected:
    (3, 1)
Got:
    (1, 1)
```
Could you look into that? Perhaps the algorithm is not deterministic with exact duplicates?

**Comment by rgommers** [comment]
> I see that there is still a CI failure, how am I able to see what this is as you did above?

You click on "Details" next to the failed CI run:

image

In this case, it was an unrelated failure on PowerPC, you're all good.

**Review by rgommers — APPROVED** [review]

**Comment by rgommers** [comment]
Merged, thanks @nwjenkins!

> This is my first pull request, so I am just learning the process. Please let me know if there is anything I should be doing better in the future (or now!). Thanks!

You did it all correctly:)

**Comment by mdhaber** [comment]
Thanks @nwjenkins. Would you like your name to be added to [THANKS.txt](https://github.com/scipy/scipy/blob/master/THANKS.txt)? If so, please make an entry [here](https://github.com/scipy/scipy/wiki/*THANKS.txt-additions-modifications-for-1.5.0*).

**Pull request merged**